src/proto/streams/flow_control.rs

```diff
@@ -80,14 +89,14 @@ impl FlowControl {
     ///
     /// This represents pending outbound WINDOW_UPDATE frames.
     pub fn unclaimed_capacity(&self) -> Option<WindowSize> {
-        let available = self.available as i32;
+        let available = self.available;
 
         if self.window_size >= available {
             return None;
         }
 
-        let unclaimed = available - self.window_size;
-        let threshold = self.window_size / UNCLAIMED_DENOMINATOR * UNCLAIMED_NUMERATOR;
+        let unclaimed = available.0 - self.window_size.0;
```

Comment:
If we are adding new types, maybe this could be a fn on the new type. Just an idea, I don't know if it makes sense, it also isn't important so feel free to disregard.